demo

```diff
@@ -67,4 +75,56 @@ exec docker-compose \
      -f docker-compose.storage.minio.yml \
      -f docker-compose.demo.yml \
      $CLIENT \
-     "$@"
+     "$@" &
+
+DC_PID=$!
+
+MENDER_SERVER_USER='myusername@example.com:mysecretpassword'
+MENDER_SERVER_URI='https://localhost'
+# Block until the useradm service returns an HTTP response.
+echo "Blocking until the useradm service is up!"
```

Comment:
The echo's are all here for debugging purposes, I guess they should be removed before the final PR. However, they do help follow the flow of the program here.